msteams-platform/messaging-extensions/how-to/search-commands/define-search-command.md

```diff
@@ -124,8 +124,131 @@ For the complete app manifest, see [App manifest schema](~/resources/schema/mani
+
+**Example**
+
+The following is an example of `userIds` array in the `refresh` property:
+
+```json
+    {
+        "type": "AdaptiveCard",
+        "refresh": {
+            "userIds": [
+                "<8:orgId:<AADID>>",
+                "29:<id>"
+            ],
+            "action": {
+                "type": "Action.Execute",
+                "data": {}
+            }
+        },
+        "body": [
+            {
+                "type": "TextBlock",
+                "text": "Hello World!",
+                "wrap": true
+            }
+        ],
+        "actions": [
+            {
+                "type": "Action.Execute",
+                "data": {},
+                "title": "Hello"
+            }
+        ]
+    }
+```
+> [!NOTE]
+> Automatic refresh is enabled for all users in the group chat or channel with more than or equal to 60 users. For conversations (group chat or channel) with more than 60 users, you can provide a manual refresh button or use the refresh button in the message options menu to get the latest result.
+ 
+```json
+    {
+        "type": "AdaptiveCard",
+        "refresh": {
+            "action": {
+                "type": "Action.Execute",
+                "data": {}
+            }
+        },
+        "body": [
+            {
+                "type": "TextBlock",
+                "text": "Hello World!",
+                "wrap": true
+            }
+        ],
+        "actions": [
+            {
+                "type": "Action.Execute",
+                "data": {},
+                "title": "Hello"
+            }
+        ]
+    }
+```
+
+### Just-In Time (JIT) install
+
+When a user selects a card and sends it in a group chat or channel, a **Just-In Time (JIT)** installation prompt appears. After the user selects the **send** option, the bot installs the card in the background for all the users in the group.
+
+> [!NOTE]
+> For apps that don’t have `Action.Execute` and `refresh` schema defined, the install prompt is not shown to the users.
+
+The following is an example of a dynamic scenario for JIT install:
+
+1. The users selects a card in a group chat or channel.
+
+    :::image type="content" source="../../../assets/images/messaging-extension/universal-actions-jit-post-card.jpg" alt-text="user selects card":::
+
+1. The card is posted in to the compose box and a JIT installation prompt appears.
+
+    :::image type="content" source="../../../assets/images/messaging-extension/universal-actions-jit-install-prompt.jpg" alt-text="app shows installation prompt":::
+
+1. After the user selects send, the bot installs the card in the background for all the users in the group.
+
+    :::image type="content" source="../../../assets/images/messaging-extension/universal-actions-jit-sent.jpg" alt-text="card is sent to users":::
+
```

Comment:
Numbering needs to be fixed here.

I would suggest that we have images for both mobile and desktop the way we have here -https://docs.microsoft.com/en-us/microsoftteams/platform/task-modules-and-cards/cards/universal-actions-for-adaptive-cards/overview?tabs=mobile